src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -308,3 +308,39 @@ void medToolBox::toXMLNode(QDomDocument* doc, QDomElement* currentNode)
 	currentNode->appendChild(elmt);
 }
 
+void medToolBox::handleDisplayError(int error)
+{
+    switch (error)
+    {
+    case medAbstractProcess::PIXEL_TYPE:
+        displayPixelTypeError();
+        break;
+    case medAbstractProcess::DIMENSION:
+        displayDataDimensionError();
+        break;
+    case medAbstractProcess::MESH_TYPE:
+        displayMeshTypeError();
+        break;
+    }
+}
+
+void medToolBox::displayPixelTypeError()
+{
+    QString error = "Pixel type not yet implemented";
+    qDebug() << name() + ": " + error;
+    medMessageController::instance()->showError(error,3000);
+}
+
+void medToolBox::displayDataDimensionError()
```

Comment:
As I read it, medAbstractProcess::DIMENSION could apply to any error related to dimension, not necessarily that the dimension should be 3. The error and functions names should be more explicit.